### ISSUE TITLE
Upgrade Apollo

### ISSUE THREAD

**Issue by rnystrom** [rnystrom/GitHawk#367]
This broke a lot of things last time I tried. Let's give it a go again.

**Comment by Sherlouk** [comment]
Yeah... that's rough 😅

**Comment by Sherlouk** [comment]
Also just looked at the type check warnings that are new in Xcode 9, and Apollo has one "expression took **15 seconds** to type-check". If they update their code to use new Swift 4 multiline strings, it should massively help the compiler!

**Comment by BasThomas** [comment]
Holy cow.

**Comment by Sherlouk** [comment]
Yeah kind of stupid, all of those massive concatenated strings they have for the schemas. In the create issue PR I've enabled all the flags on debug builds with a 200ms limit. Not particularly sensitive but should draw our attention to any big mistakes!

**Comment by rnystrom** [comment]
@Sherlouk probably worth [filing an issue](https://github.com/apollographql/apollo-ios)?

**Comment by Sherlouk** [comment]
Raised https://github.com/apollographql/apollo-ios/issues/152

**Comment by Sherlouk** [comment]
Apollo guys have closed the issue above with a fix on the latest beta, if we upgrade should have some improvements to compile time!

**Comment by Sherlouk** [comment]
So very much still WIP on this, they've definitely made some improvements to the structure of files and requests but equally some bizarre changes. Fragments are a lot more annoying to work with, but this is something that we can probably re-address once I've actually got something building!

It's not a very intuitive process I'm finding
